FBAllocationTracker/NSObject+FBAllocationTracker.mm

```diff
@@ -46,3 +48,26 @@ - (void)fb_newDealloc
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
+
+
+bool fb_isZombieObject(_Nullable id object) {
+  if (object == nil) { return true; }
+  
+  const char *className = object_getClassName(object);
+  static const char *zombiePrefix = "_NSZombie";
+  return strncmp(className, zombiePrefix, strlen(zombiePrefix)) == 0;
+}
+
+bool fb_isNSZombieEnabled(void) {
+  static bool onceFlag = false;
+  static bool zombieEnabled = false;
+  
+  if (!onceFlag) {
```

Comment:
It won't change over the course of one app run, please use `dispatch_once` for that :)